requirements.txt

```diff
@@ -15,7 +15,7 @@ Flask-Migrate==2.7.0
 git+https://github.com/mitsuhiko/flask-sqlalchemy.git@500e732dd1b975a56ab06a46bd1a20a21e682262#egg=Flask-SQLAlchemy==2.3.2.dev20190108
 Flask==1.1.2
 click-datetime==0.2
-eventlet==0.30.2
+eventlet==0.31.0
```

Comment:
This is upgraded by this PR https://github.com/cds-snc/notification-api/pull/1376